spec/requests/admin/users_controller_spec.rb

```diff
@@ -149,6 +149,21 @@
       expect(log.details).to match(/because I said so/)
     end
 
+    it "checks if user is suspended" do
+      put "/admin/users/#{user.id}/suspend.json", params: {
+        suspend_until: 5.hours.from_now,
+        reason: "because I said so"
+      }
+
+      put "/admin/users/#{user.id}/suspend.json", params: {
+        suspend_until: 5.hours.from_now,
+        reason: "because I said so too"
+      }
+
+      expect(response.status).to eq(409)
+      expect(response.parsed_body["message"]).to eq("User was already suspended by #{admin.username} just now.")
```

Comment:
We try to avoid testing against the value of a translation as the test then become tied to the translated string. Instead, you should call `I18n.t()`

```suggestion
      expect(response.parsed_body["message"]).to eq(I18n.t("user.already_silenced", ...))
```

Reply:
It was the "just now" part which gave me a little trouble, but I figured that out. Thanks!